.github/ISSUE_TEMPLATE/add_conference.yml

```
@@ -59,9 +59,9 @@ body:
   - type: textarea
     id: conference_description
     attributes:
-      label: Conference Description
-      description: Add any pertinent information about the conference e.g. focuses, topics, talk-length
-      placeholder: "This is a general python conference with open spaces and sprints"
+      label: Summary
+      description: Summary of the news you're sharing about this conference.
+      placeholder: "PyCon has been announced alongside it's [website](url) and [cfp]. The conference is 27-31 April with workshops and sprints"
```

```suggestion
      placeholder: "PyCon US has been announced alongside it's [website](url) and [cfp]. The conference is 27-31 April with workshops and sprints"
```

Just a minor change to be accurate with the title of the conference. Or, alternatively, we can be generic with something like

```suggestion
      placeholder: "ExampleCon has been announced alongside it's [website](url) and [cfp]. The conference is 27-31 April with workshops and sprints"
```